src/include/daos_srv/daos_server.h

```diff
@@ -186,6 +205,51 @@ struct dss_drpc_handler {
 	drpc_handler_t	handler;	/** dRPC handler for the module */
 };
 
+/* The profile structure to record single operation */
+struct srv_profile_op {
+	int		pro_id;		/* id in obj_profile_op */
+	uint64_t	pro_time;	/* time cost for this id */
+};
+
+/* The chunk for a group of srv_profile */
+struct srv_profile_chunk {
+	d_list_t	      spc_chunk_list;
+	struct srv_profile_op *spc_profiles;
+	int		      spc_idx;
+	int		      spc_chunk_size;
+};
+
+#define MAX_PROFILE_OP	64
+/* Holding the total trunk list for a specific profiling module */
+struct srv_profile {
+	struct srv_profile_chunk *sp_current_chunk;
+	uint64_t	sp_time[MAX_PROFILE_OP];
+	d_list_t	sp_list;	/* active list for profile chunk */
+	d_list_t	sp_idle_list;	/* idle list for profile chunk */
+	/* Count in idle list & list */
+	int		sp_chunk_total_cnt;
+	/* count in list */
+	int		sp_chunk_cnt;
+	char		*sp_dir_path;	/* Where to dump the profiling */
+	char		**sp_names;	/* profile name */
+	ABT_thread	sp_dump_thread;	/* dump thread for profile */
+	int		sp_stop:1;
+};
+
+struct dss_module_ops {
+	/* The callback for each module will choose ABT pool to handle RPC */
+	ABT_pool (*dms_abt_pool_choose_cb) (crt_rpc_t *rpc, ABT_pool *pools);
```

Comment:
(style)  space prohibited between function name and open parenthesis '('
(style)  need consistent spacing around '*' (ctx:WxV)

Reply:
I think this is just complaining about the space between the function pointer name and the parameter list

e.g. use int (*x)(int x) instead of int (*x) (int x)